dist/openrc/zrepl

```diff
@@ -0,0 +1,14 @@
+#!/sbin/openrc-run
+
+command='/usr/local/bin/zrepl'
```

Comment:
Why not `/usr/bin/zrepl` ? That's where the .deb and .rpm packages install it.

Reply:
@problame I've just copied the path from the [systemd service](https://github.com/zrepl/zrepl/blame/master/dist/systemd/zrepl.service#L8) already present in this repo.

Also, if someone is copying the service from this repository instead of using the one provided by the distro it probably means zrepl was installed manually so it should be located in /usr/local/bin.

Reply:
Allright.